src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="container">
+    <div class="calendar">
```

Comment:
Calendar should have also two modifiers:
**calendar--start-day--sun** - sun can be a default
**calendar--month-length--2**9 - 29 can be default value
**block-name--modifier-name--modifier-value**